backend/database/models/incident.py

```diff
@@ -86,58 +113,24 @@ class Incident(db.Model, CrudMixin):
     # case_id = db.Column(db.Integer)  # TODO: foreign key of some sort?
 
 
-# Association tables
-
-
-# class IncidentVictim(db.Model):  # TODO: rename as IncidentVictimXRef ?
-#     id = db.Column(db.Integer, primary_key=True)
-#     incident_id = db.Column(db.Integer)  # TODO: foreign key
-#     victim_id = db.Column(db.Integer)  # TODO: foreign key
-#
-#
-# class IncidentDescription(db.Model):
-#     # (Note) Removed all the `description_*` prefixes.
-#     id = db.Column(db.Integer, primary_key=True)  # description id
-#     incident_id = db.Column(db.Integer)  # TODO: foreign key
-#     text = db.Column(db.Text)
-#     type = db.Column(db.Text)  # TODO: enum
-#     # TODO: are there rules for this column other than text?
-#     source = db.Column(db.Text)
-#
-#
-# class IncidentAction(db.Model):
-#     id = db.Column(db.Integer, primary_key=True)  # action id
-#     incident_id = db.Column(db.Integer)  # TODO: foreign key
-#     date = db.Column(db.DateTime)
-#     action = db.Column(db.Text)  # TODO: Not sure what this is.
-#     actor = db.Column(db.Text)  # TODO: Not sure what this is.
-#     notes = db.Column(db.Text)
-#
-#
-# class IncidentTag(db.Model):
-#     # TODO: rename as IncidentTagXRef ?
-#     id = db.Column(db.Integer, primary_key=True)
-#     incident_id = db.Column(db.Integer)  # TODO: foreign key
-#     tag_id = db.Column(db.Integer)  # TODO: foreign key
-#
-#
-# class IncidentUseOfForce(db.Model):
-#     # TODO: rename to IncidentUseOfForceXRef ?
-#     id = db.Column(db.Integer, primary_key=True)
-#     incident_id = db.Column(db.Integer)  # TODO: foreign key
-#     use_of_force_id = db.Column(db.Integer)  # TODO: foreign key
-#
-#
-# class ResultOfStopAtIncident(db.Model):
-#     # TODO: Rename to IncidentResultOfStopXRef?
-#     # TODO:
-#     #  Does this need to be an crossref table?
-#     #  Is there a many-to-many relationship?
-#     id = db.Column(db.Integer, primary_key=True)
-#     incident_id = db.Column(db.Integer)  # TODO: foreign key
-#     result_of_stop_id = db.Column(db.Integer)  # TODO: foreign key
-#
-#
-# class ResultOfStop(db.Model):
-#     id = db.Column(db.Integer, primary_key=True)
-#     result = db.Column(db.Text)
+class Action(db.Model):
+    id = db.Column(db.Integer, primary_key=True)  # action id
+    incident_id = db.Column(
+        db.Integer, db.ForeignKey("incident.id"), nullable=False
+    )
+    date = db.Column(db.DateTime)
+    action = db.Column(db.Text)  # TODO: Not sure what this is.
+    actor = db.Column(db.Text)  # TODO: Not sure what this is.
+    notes = db.Column(db.Text)
+
+
+class UseOfForce(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
+    item = db.Text()
+
+
+class ResultOfStop(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
+    result = db.Column(db.Text)
```

Comment:
@dwreeves should each of these get its own file? They're all one to many with incident

Reply:
oh good catch. Uhhh.... probably.